.github/workflows/e2e-master.yml

```diff
@@ -136,6 +136,13 @@ jobs:
           --ci-build-id "${{ github.run_id }}-${{ github.run_attempt }}"
       env:
         TERM: xterm
+    - name: Run Cypress tests on ${{ matrix.folder }}
```

Comment:
Should this have a different name? I'm not sure how Github handles workflows with the same name - I'll defer to @ariya

Reply:
It's only for display purpose, shouldn't cause conflict or any terrible issue.